apps/web/package.json

```diff
@@ -25,6 +25,7 @@
     "@trpc/react-query": "11.0.0-rc.441",
     "@trpc/server": "11.0.0-rc.441",
     "dayjs": "^1.11.13",
+    "fuse.js": "^7.0.0",
```

Comment:
Might want to remove the `fuse.js` dependency here since we won't be doing fuzzy searching on the web app anymore. We will add `fuse.js` to the `api` package and use it in the endpoints there.